src/system_tests/load/tests/test_isolation.py

```
@@ -0,0 +1,81 @@
+        get_chat_messages_count=_messages,
+    )
+    assert ok is False
+    assert "6 variables" in error or "found 6" in error
+
+
+def test_validate_threads_state_uniformity_requires_matching_counts():
+    snapshots = [
+        ThreadStateSnapshot(0, "a", 3, 6),
+        ThreadStateSnapshot(1, "b", 3, 6),
+    ]
+    ok, error, report = validate_threads_state_uniformity(
```

Ruff (0.15.15)

[warning] Unpacked variable `error` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)